<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>380</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="Select">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select Your Muscle Group(s) Checklist:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="groupNumber">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>265</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="workoutPer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string># Workouts Per Group:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Workout!</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="shouldersCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SHOULDERS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bicepsCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BICEPS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chestCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CHEST</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="legsCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LEGS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="backCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="absCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ABS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="tricepsCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TRICEPS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="calvesCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CALVES</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cardioCheck">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>105</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CARDIO</string>
    </property>
   </widget>
   <widget class="QLabel" name="CheckList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>[CHECKLIST]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
